### DISCUSSION TITLE
Error with web GUI

### DISCUSSION THREAD

**Discussion by Giro100** [neonbjb/tortoise-tts#639] [General]
when generating a line, i get an error msg with no explaining. the file does generate but it takes twice the time, and i dont see anything in the web gui
![image](https://github.com/neonbjb/tortoise-tts/assets/110100068/5ee4c7c5-c518-4da3-86a2-1197803739aa)
![image](https://github.com/neonbjb/tortoise-tts/assets/110100068/5ed12eca-fdf0-4b6a-83ee-28a120d8e2d2)

**Comment by fakerybakery** [comment]
Your issue is related to the MRQ repo, not this repository. Please open an issue on the [MRQ repo](https://git.ecker.tech/mrq/ai-voice-cloning) instead. Thank you

**Comment by DevangPatelR** [comment]
Moved to [JarodMica ai-voice-cloning](https://github.com/JarodMica/ai-voice-cloning)
